cardano-constitution/README.md

```diff
@@ -193,3 +200,30 @@ the union (Sum) of the ledger guarantees required by all our current implementat
 - `src/Cardano/Constitution/Validator/Common.hs` : common code between the 2 engines
 - `data/*` : contains the JSON configuration files
 - `test/*`: testing code
+- `app/create-json-envelope`: an executable to construct constitution script, ready to be submitted to the chain
+
+## Building the script
+
+We have created an executable `create-json-envelope` to ease the creation of a constitution script.
+This executable is disabled by default; to enable it, uncomment the related lines inside this repo's `cabal.project`:
```

Comment:
Is there a future plan to move this out of the repo?

Reply:
Yes